recipes/tensorflowjs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "tensorflowjs" %}
+{% set version = "3.11.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/py3/t/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
+  sha256: cbdfa5da24987229e14bb836fe6db5ad0eba3860da7c9d6890968163eab2358c
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
```suggestion
  noarch: python
```

Reply:
It cannot build on windows because there are not any tensorflow versions >2.0 that support a windows build: https://github.com/conda-forge/tensorflow-feedstock/blob/master/recipe/meta.yaml

Reply:
That's fine. `noarch` packages only need to pass on Linux on CI.

Reply:
ah okay...will update, thanks!